packages/kbn-optimizer/limits.yml

```diff
@@ -117,3 +117,4 @@ pageLoadAssetSize:
   dataViewManagement: 5000
   reporting: 57003
   visTypeHeatmap: 25340
+  screenshotting: 17017
```

Comment:
Do you mind helping me understand the value of `screenshotting` adding a new 2kb request to every page load? It looks like it stored a very tiny amount of state which could easily be managed by the reporting plugin and just increases the overall page load size and request count, but I'm probably missing something.

Reply:
If there isn't a good reason it would be great if we could set `"ui": false` to the `kibana.json` file and move the `contextStorage` stuff to the `reporting` plugin.

Reply:
That exposes a generic service working with the screenshotting context. The idea is that some of the plugins will use that to extract some serialized data without having a dependency on the reporting. This code will be merged with the `screenshotMode` plugin later, so won't stay long here.